village/src/components/Smurfs.js

```diff
@@ -1,9 +1,31 @@
 import React, { Component } from 'react';
-
+import axios from 'axios';
 import Smurf from './Smurf';
 
 class Smurfs extends Component {
   // add any needed code to ensure that the smurfs collection exists on state and it has data coming from the server
+  constructor() {
+    super();
+    this.state = {
+      smurfs: [],
+      name: '',
+      age: '',
+      height: ''
+  }
+
```

Comment:
Try changing the Smurf component into a functional component without its own state.  Try to figure out how to pass these values from the state to this component to be rendered out.